### PR TITLE
Enable support of newest Ubuntu 21.04 and 20.04 (AO-17661)

### PR DESCRIPTION
**Pull request by d-maslyk**
Changes:
- Enable support of newest Ubuntu 21.04 and 20.04 
- - align molecule test to Snap in 4.1.0 version (embedded  plugins)

### CODE REVIEW

File: .yamllint [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (lootek) [inline .yamllint:1]: <3

File: CODEOWNERS [hunk 2]
```diff
@@ -1 +1 @@
-* @d-maslyk @tomekkolo @swi-snap-agent
+* @swisnap-krk
```
Comment (lootek) [inline CODEOWNERS:1]: 👍

File: molecule/complex_configuration/tests/test_complex_configuration.py [hunk 3]
```diff
@@ -31,10 +31,10 @@ def test_swinsapd_service(host):
 
 def test_process_swisnapd(host):
     assert host.process.get(user=SOLARWINDS, comm=SWISNAPD)
-    # checking collector plugins: aosystem, processes, logs, bridge, apache
-    assert len(host.process.filter(user=SOLARWINDS, comm="snap-plugin-col")) == 5
-    # checking default publisher plugins: publisher-appoptics v1, publisher-appoptics v2, publisher-processes
-    assert len(host.process.filter(user=SOLARWINDS, comm="snap-plugin-pub")) == 3
+    # checking collector plugins: logs, bridge, apache
+    assert len(host.process.filter(user=SOLARWINDS, comm="snap-plugin-col")) == 3
+    # checking default publisher plugins: publisher-appoptics v1, publisher-appoptics v2
```
Comment (lootek) [inline molecule/complex_configuration/tests/test_complex_configuration.py:36]: well, publisher appoptics v2 is inprocess, should not appear on the processes list

Comment (d-maslyk) [reply molecule/complex_configuration/tests/test_complex_configuration.py:36]: As talked offline, this is expected.

**Review by lootek — APPROVED** [review]

**Pull request merged**